### PR TITLE
fix(config): add value 3 to param 4 for Zooz Zen16

### LINKED ISSUE #4403: Zooz ZEN16 Missing Configuration Value "3" for Parameter 4
### Is your problem within Home Assistant (Core or Z-Wave JS Integration)?

NO, my problem is NOT within Home Assistant or the ZWave JS integration

### Is your problem within ZWaveJS2MQTT?

NO, my problem is NOT within ZWaveJS2MQTT

### Checklist

- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there.

### Describe the bug

What causes the bug?
Try to set "Switch 3: Type" (parameter number 4) to "Garage door" in the same way that Switches 1 and 2 are done. The dropdown for Switch 3 does not include "Garage door", even though the manufacturer documentation indicates that it is available.

What do you observe?
"Garage door" value does not exist in the dropdown list for Switch 3.

What did you expect to happen?
"Garage door" value exists in the dropdown list for Switch 3, identical to how it's seen in for Switch 1 and Switch 2.

  1. In Home Assistant settings for the zen16 device, go to the Configure Device link
  2. Click the dropdown box for parameter 4 - "Switch 3: Type"
  3. Note that "Garage door" is not in the list (this is value "3" in the documentation)

### Device information

Manufacturer: Zooz
Model name: ZEN16
Node ID in your network: 110

### How are you using `node-zwave-js`?

- [ ] `zwavejs2mqtt` Docker image (latest)
- [ ] `zwavejs2mqtt` Docker image (dev)
- [ ] `zwavejs2mqtt` Docker manually built _(please specify branches)_
- [ ] `ioBroker.zwave2` adapter _(please specify version)_
- [X] `HomeAssistant zwave_js` integration  _(please specify version)_
- [ ] `pkg`
- [ ] `node-red-contrib-zwave-js` _(please specify version, double click node to find out)_
- [ ] Manually built from GitHub _(please specify branch)_
- [ ] Other _(please describe)_

### Which branches or versions?

version:
`node-zwave-js` info from HA:
Driver Version: 8.11.9
Server Version: 1.15.0

### Did you change anything?

no

### If yes, what did you change?

_No response_

### Did this work before?

No, it never worked anywhere

### If yes, where did it work?

_No response_

### Attach Driver Logfile

Adding log info will provide no useful info given the scenario.

### PR DESCRIPTION
**Pull request by GaTechThomas**
Fixes #4403.

### CODE REVIEW

File: packages/config/config/devices/0x027a/zen16.json [hunk 1]
```diff
@@ -175,6 +175,32 @@
 		},
 		{
 			"#": "4",
+			"$if": "firmwareVersion <= 1.1",
+			"label": "Switch 3: Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen16.json]:
```suggestion
			"maxValue": 2,
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Small typo, I'll fix this myself

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks!

**Comment by GaTechThomas** [comment]
Good catch on the maxValue. On that point, param 2 and param 3 firmware <= 1.1 both have maxValue 3 when it should be 2. Is that a significant item? This was existing in the previous version of the code - should it be a new issue?

**Comment by AlCalzone** [comment]
Yeah, these should be corrected. Wonder why our linter doesn't catch those.

**Pull request merged**